jsapp/scss/components/_kobo.form-view.scss

```diff
@@ -462,7 +462,7 @@
 
     i {
       color: $kobo-orange;
-      font-size: 32px;
```

Comment:
Not related to this line but rather the line in this change: https://github.com/kobotoolbox/kpi/commit/47ae961b8f#diff-00bdd4e8caa9bf2acbfe587c2dff999ef359227e43f94e821434d5968d22e8d8R99

We made the change so that the "Connect Projects" tab looked vertically aligned with its icon (as the text spans two lines):
![image](https://user-images.githubusercontent.com/16762675/127393807-0d499cfe-eb27-44a2-bcc1-e1746d6d996f.png)

but now it looks like that's messing up the vertical alignment of the other tabs.

I think we should add a custom modifier for only long titled tabs to use `display: flex`

Reply:
Right, I added `align-items: center` to it and it's perfect :)